AdyenCard/Form/FormCardExpiryDateItem.swift

```diff
@@ -74,7 +74,7 @@ internal final class FormCardExpiryDateItem: FormTextItem {
 }
 
 extension FormItemViewBuilder {
-    internal func build(with item: FormCardExpiryDateItem) -> FormItemView<FormCardExpiryDateItem> {
-        FormTextItemView<FormCardExpiryDateItem>(item: item)
+    internal func build(with item: FormCardExpiryDateItem) -> FormItemView<FormTextInputItem> {
```

Comment:
```suggestion
    internal func build(with item: FormCardExpiryDateItem) -> FormItemView< FormCardExpiryDateItem > {
```

Reply:
As discussed in person, it has to do with the way Swift handles type inheritance with generic parameters. In this case, even if `FormCardExpiryDateItem` is a child of `FormTextInput`, Swift forces you to keep the parent class.|